data/data/powervs/bootstrap/variables.tf

```diff
@@ -1,7 +1,10 @@
-variable "bootstrap" {}
-variable "cloud_instance_id" {}
-variable "image_name" {}
-variable "network_name" {}
-variable "proc_type" {}
-variable "sys_type" {}
-variable "cluster_id" {}
+variable "powervs_bootstrap" {}
```

Comment:
I feel inside module variables can be a simple one,! only variables we need to have some prefix is the one which are exposed in the root dir

Reply:
Ahhhh - I noticed that the loadbalancer did that after I uploaded, will fix that now